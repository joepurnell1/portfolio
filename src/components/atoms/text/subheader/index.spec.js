import React from 'react';
import renderer from 'react-test-renderer';
import Header from './index';

describe('Subheader component', () => {
  it('should render default as expected', () => {
    const render = renderer.create(
      <Header>You don&apos;t need nobody else, and I know that you still believe</Header>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });

  it('should render with custom colour as expected', () => {
    const render = renderer.create(
      <Header colour="#F00B42">It&apos;s all yours</Header>,
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});
