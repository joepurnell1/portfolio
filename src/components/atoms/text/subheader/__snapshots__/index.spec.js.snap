// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subheader component should render default as expected 1`] = `
<h2
  className="sc-bdVaJa fxUZL"
>
  You don't need nobody else, and I know that you still believe
</h2>
`;

exports[`Subheader component should render with custom colour as expected 1`] = `
<h2
  className="sc-bdVaJa heXFlR"
>
  It's all yours
</h2>
`;
